README.md

```diff
@@ -97,12 +102,13 @@ which is the front-end pattern library used in this project.
 
 
 ## Open source licensing info
+
 1. [TERMS](TERMS.md)
 2. [LICENSE](LICENSE)
 3. [CFPB Source Code Policy](https://github.com/cfpb/source-code-policy/)
 
 
 ## Credits and references
 
-This project uses the [Capital Framework](https://github.com/cfpb/capital-framework)
-for its user interface and layout components.
+This project uses [Capital Framework](https://github.com/cfpb/capital-framework)
+as the bass of its user interface and layout components.
```

Comment:
```suggestion
as the basis of its user interface and layout components.
```

🎸 🎵 

More seriously, should this be updated to point to the [Design System](https://cfpb.github.io/design-system/) instead?

Reply:
Hah! My sub(woofer)consciousness at work? But I think we should hold off on mentioning the Design System until cfgov-refresh is pulling in the packages from it.